### PR TITLE
Update python-vdf and protontricks for new Steam vdf format

### PR DESCRIPTION
**Pull request by Staudey**
**Summary**

This updates `python-vdf` and `protontricks` so they are compatible with the new vdf format in the current Steam beta

**Test Plan**

Used `protontricks` to install components into a game's proton prefix.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/py/python-vdf/package.yml
Comment (TraceyC77) [inline packages/py/python-vdf/package.yml:1]: This package fails to build on my system, looks like there's a missing dependency

```
~/YPKG/root/python-vdf/build/py3build ~/YPKG/root/python-vdf/build ~/YPKG/root/python-vdf/build/vdf-3.4
+ [[ -f setup.py ]]
+ python3 setup.py install --root=/home/build/YPKG/root/python-vdf/install
Traceback (most recent call last):
  File "/home/build/YPKG/root/python-vdf/build/py3build/setup.py", line 6, in <module>
    import vdf
ModuleNotFoundError: No module named 'vdf'
+ exit
Command '['/bin/bash', '--norc', '--noprofile', '/tmp/ypkg-installj894rq83']' returned non-zero exit status 1.
[Build] install failed for python-vdf
 ✗  Failed to build packages err="Failed to start build of package, reason: exit status 1\n"
task: Failed to run task "build": exit status 1
```

**Review by TraceyC77 — CHANGES_REQUESTED** [review]
python-vdf doesn't build

File: packages/py/python-vdf/package.yml
Comment (Staudey) [reply packages/py/python-vdf/package.yml:1]: That is weird, it still builds perfectly fine for me. I hope this is not another of these weird _fakeroot_ issues.

Comment (HarveyDevel) [reply packages/py/python-vdf/package.yml:1]: This builds fine for me too. Going to try landing it.

**Review by HarveyDevel — APPROVED** [review]
LGTM, thanks!

**Pull request merged**